rpcs3/Emu/CPU/CPUThread.h

```diff
@@ -296,6 +301,7 @@ class cpu_thread
 
 private:
 	static thread_local cpu_thread* g_tls_this_thread;
+	const thread_class m_class;
```

Comment:
Dont add or remove any member of cpu_thread, existing PPU LLVM cache breaks with this change.

Reply:
then the cache seems to need a redesign

Reply:
Maybe, but that has nothing to do with this pr though. You can simply assign enum values to the previously magic numbers and call it a day.